src/tools/GraphemeTableGen/Program.cs

```diff
@@ -0,0 +1,579 @@
+using System.Text;
+using System.Runtime.InteropServices;
+using System.Numerics;
+using System.Xml.Linq;
+using TrieType = uint;
+
+// UAX #29 uses "A ÷ B" to indicate that there's a potential break opportunity between A and B.
+// But ÷ is not a valid identifier in Go, so we use Ω which is.

```

Comment:
> not a valid identifier in Go

good thing this is C#!

Reply:
personally I think Ω looks WAY TOO SIMILAR to 0 - but then, syntax highlighting would make it better